SConstruct

```diff
@@ -937,7 +937,7 @@ def TryScript(config,text,executable):
 
     # Run the given executable with the source file we just built
     output = config.sconf.confdir.File(f + '.out')
-    node = config.env.Command(output, source, executable + " < $SOURCE >& $TARGET")
+    node = config.env.Command(output, source, executable + " $SOURCE > $TARGET")
```

Comment:
Oops.  I hadn't meant to commit this at all - this PR really is an embarrassment.  That said, I'm glad it's fixed downstream, because I did indeed have to make that change to get it working on my machine.  Removing the `<` was non necessary for that; I was just something I tried while trying to figure out what went wrong.

I'll remove this change entirely before I rebase.